catalog/admin/admin-poller-service/src/main/java/org/codice/ddf/catalog/admin/poller/AdminPollerServiceBean.java

```diff
@@ -215,6 +252,106 @@ public int compare(Map<String, Object> o1, Map<String, Object> o2) {
         return metatypes;
     }
 
+    @Override
+    public List<Serializable> updatePublications(String source, List<String> destinations)
```

Comment:
You should probably check destinations for null since this is exposed as an mbean.